tests/installation/module_registration/register_extensions_and_modules.pm

```diff
@@ -11,12 +11,24 @@
 use base 'y2_installbase';
 use strict;
 use warnings;
-use testapi qw(save_screenshot get_required_var);
+use testapi qw(save_screenshot get_required_var get_var);
 
 sub run {
     my @scc_addons = grep($_, split(/,/, get_required_var('SCC_ADDONS')));
     $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
     save_screenshot;
```

Comment:
neat: you could use a extra blank line between logical section of the code, in general after save_screenshot as a way to style your code in general. It is much clear to read.

Reply:
save_screenshot goes after the logical section, then you can introduce a blank line to separate logical section:
- logical section
- save_screenshot
- blank line
The code doesn't follow that, but it is just a suggestion, we can follow in the future.